.editorconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) James May, <fowl@fowlsmurf.net>, et al.
+# SPDX-License-Identifier: curl
+
+# EditorConfig: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+[*]
```

Comment:
This section contradicts `.gitattributes` which lists several extensions that use crlf.

Maybe using `[*.md]` here would be safer?